packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -895,6 +895,28 @@ const TESTS: [string, any, Record<string, any>?][] = [
     ]],
   ])],
 
+  ['passwordForgotOtpEmail', new Map<string, Test | any>([
+    ['subject', { test: 'equal', expected: 'Forgot your password?' }],
+    ['headers', new Map([
+      ['X-Password-Forgot-OTP', { test: 'equal', expected: MESSAGE.code}],
+      ['X-SES-MESSAGE-TAGS', { test: 'equal', expected: sesMessageTagsHeaderValue('passwordForgotOtp') }],
+      ['X-Template-Name', { test: 'equal', expected: 'passwordForgotOtp' }],
+      ['X-Template-Version', { test: 'equal', expected: TEMPLATE_VERSIONS.passwordForgotOtp }],
+    ])],
+    ['html', [
+      { test: 'include', expected: 'Forgot your password?' },
+      { test: 'include', expected: 'We received a request for a password change on your Mozilla account from:' },
+      { test: 'include', expected: MESSAGE.code },
+      { test: 'include', expected: `${MESSAGE.device.uaBrowser} on ${MESSAGE.device.uaOS} ${MESSAGE.device.uaOSVersion}` },
+    ]],
+    ['text', [
+      { test: 'include', expected: 'Forgot your password?' },
+      { test: 'include', expected: 'We received a request for a password change on your Mozilla account from:' },
+      { test: 'include', expected: MESSAGE.code },
+      { test: 'include', expected: `${MESSAGE.device.uaBrowser} on ${MESSAGE.device.uaOS} ${MESSAGE.device.uaOSVersion}` },
+    ]],
```

Comment:
Should we add the expectation for the automatedEmail block?

Reply:
🤷 

I can add it if I'm going to make other changes to the PR.